pkg/scheduler/preemption/preemption.go

```diff
@@ -432,19 +431,19 @@ func (p *Preemptor) runFirstStrategy(preemptionCtx *preemptionCtx, candidates []
 			}
 		}
 	}
-	return fits, targets, retryCandidates
+	return false, targets, retryCandidates
```

Comment:
Ah, I see, this is tricky. Before it was possible to have `fits: true`, and `len(retryCandidates) > 0` returned here.

however, we can safely return `fits: true` and `retryCandidates: false`, because the `retryCandidates` are only used for the second strategy if `fits: false`.